.gitignore

```diff
@@ -29,3 +29,6 @@ packages/devtools_test/lib/src/mocks/generated.mocks.dart
 **/test/**/failures/
 *.dill
 DEPENDENCIES.md
+
+# Ignore logs generated by VS Code
+**/logs/**
```

Comment:
Is this change intentional?

Reply:
Yeah I can break it out to a separate PR if we want, but I accidentally committed some logs that VS code created in my first commit, so just figured I'd add it while I was here.